src/ZwiftPacketMonitor.csproj

```diff
@@ -1,26 +1,30 @@
-<Project Sdk="Microsoft.NET.Sdk" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project Sdk="Microsoft.NET.Sdk" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework> <!-- netstandard2.0 -->
     <id>ZwiftPacketMonitor</id>
-    <Version>1.2.8</Version>
+    <Version>1.2.9.1</Version>
     <GeneratePackageOnBuild Condition="'$(Configuration)' == 'Release'">true</GeneratePackageOnBuild>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
-    <Authors>Brad Walker (@braddwalker)</Authors>
+    <Authors>ruffk / Brad Walker (@braddwalker)</Authors>
     <Description>This is a port of the node zwift-packet-monitor project which monitors Zwift network packets</Description>
     <Title>Captures TCP and UDP packets produced/consumed by the Zwift online cycling simulator</Title>
-    <PackageProjectUrl>https://github.com/braddwalker/ZwiftPacketMonitor</PackageProjectUrl>
-    
+    <PackageProjectUrl>https://github.com/ruffk/ZwiftPacketMonitor</PackageProjectUrl>
```

Comment:
This URLs should reflect the correct project URL -- https://github.com/braddwalker/ZwiftPacketMonitor

Reply:
Done